plugin/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -82,17 +82,123 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 		  "kind": "Policy",
```

Comment:
update the dire comment above the 1.1, 1.2, and 1.3 test data to match the 1.0 comment :)